pages/course/[code].tsx

```diff
@@ -115,7 +115,18 @@ const CoursePage: NextPage<Partial<CoursePageProps>> = ({
         <Text fontFamily="title" fontWeight="500" fontSize="2xl">
           Course Books
         </Text>
-        <BookCardGrid books={books} />
+
+        {isLoading ? (
+          <Flex width="100%" height="56" align="center" justifyContent="center">
+            <Spinner />
+          </Flex>
+        ) : !books || books.length === 0 ? (
+          <Text my="10" fontSize="lg">
+            No course books found.
+          </Text>
+        ) : (
+          <BookCardGrid books={books} />
+        )}
```

Comment:
The `isLoading` here refers to posts. I don't think this data should have a loading state.